pkg/ruler/manager.go

```diff
@@ -66,6 +58,13 @@ func NewDefaultMultiTenantManager(cfg Config, managerFactory ManagerFactory, reg
 		reg.MustRegister(userManagerMetrics)
 	}
 
+	managersTotal := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "cortex",
+		Name:      "ruler_managers_total",
+		Help:      "Total number of managers registered and running in the ruler",
+	})
+	reg.MustRegister(managersTotal)
```

Comment:
Please do not use `reg.MustRegister()` but create the metric with `promauto.With(reg).NewGauge()`